### ISSUE TITLE
How to extract features instead of logits?

### ISSUE THREAD

**Issue by GowthamGottimukkala** [OanaIgnat/i3d_keras#7]

**Comment by jahsylla** [comment]
Simple, load the model and remove the last layer something like that model.pop() is all you need.
